app/models/__init__.py

```diff
@@ -1,34 +1,37 @@
 from datetime import datetime
 
 from flask_login import LoginManager, UserMixin
-from sqlalchemy import (Column, Enum, ForeignKey, Integer, String,
-                        create_engine, MetaData)
+from geoalchemy2 import Geography
+from sqlalchemy import (
+    Boolean,
+    Column,
+    Enum,
+    ForeignKey,
+    Integer,
+    MetaData,
+    String,
+    create_engine,
+)
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import relationship, sessionmaker
-from sqlalchemy_utils import database_exists, create_database
-from geoalchemy2 import Geography
+from sqlalchemy_utils import create_database, database_exists
 
 Base = declarative_base()
 metadata = MetaData()
 DB_URL = "postgresql://postgres:admin@localhost/botinder"
 
-"""
-
-"""
-
 engine = create_engine(DB_URL, echo=True)
 if not database_exists(engine.url):
     create_database(engine.url)
     print("Utworzono bazę danych.")
 
+from app.models import Matches
+from app.models.messages import Messages
+from app.models.robots import RobotMessages, RobotProfile, UserRobot
+from app.models.user import Profile, User, UserCriteria
+

```

Comment:
I put this imports here because if i put them on the top thats what happens:

from app.models import (
ImportError: cannot import name 'Base' from partially initialized module 'app.models' (most likely due to a circular import) (D:\Programowanie, nauka, projekty, praca\Botinder projekt\Botinder\app\models\__init__.py)